recipes/_configure.rb

```diff
@@ -5,25 +5,28 @@
 #
 # Configures the Splunk system post package installation
 
-# Verify that clusters are configured
-if node['splunk']['node_type'] != :license_server && node['splunk']['config']['clusters'].empty?
-  if node['splunk']['node_type'] == :forwarder
-    Chef::Log.warn 'No cluster data bag configured, ensure your outputs are configured elsewhere.'
-  else
-    throw 'You need to configure at least one cluster databag.'
+unless node['splunk']['configure_apps_only']
```

Comment:
If we want this to be less reusable outside of the new server_install_only recipe and more of an internal implementation detail, I could move this to node.run_state instead of an attribute.

Reply:
Moved to run_state: https://github.com/cerner/cerner_splunk/pull/160/commits/13c0f592422824b153bd1d9caf4b72d0dbd8b825